internal/kubernetes/drainer.go

```diff
@@ -381,6 +387,34 @@ func (d *APICordonDrainer) Uncordon(n *core.Node, mutators ...nodeMutatorFn) err
 	return nil
 }
 
+// MarkDrain set a condition on the node to mark that the drain is scheduled.
```

Comment:
wrong comment string
```suggestion
// MarkDrainDelete removes the condition on the node that marked that a drain was scheduled.
```